### DISCUSSION TITLE
Cant get old value on edit custom view field

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#261] [Q&A (Help)] [answered]
i create custom radio button for repeatable element, this for make default contact number,
but i cant get old value on edit, (using for showing checked radio element)

i am using this to get old value , but it can't
`$current_value = old(square_brackets_to_dots($field['value'])) ??  $field['value'] ?? $field['default'] ?? '';
`this blade custom radio button
![image](https://user-images.githubusercontent.com/89818695/167181948-82ad58a2-9d22-41f2-b13f-139d05717fb7.png)
in controller
![image](https://user-images.githubusercontent.com/89818695/167182320-c82e5d33-762a-4f7a-b864-cb993390e458.png)
![image](https://user-images.githubusercontent.com/89818695/167182461-b50fb61f-967d-4798-a9ae-8e340202ada1.png)

when edit and adding new repeatable elements,i got default value `on`,the default value should be `false`, like when on create
![image](https://user-images.githubusercontent.com/89818695/167184097-476c46fd-0093-463b-8a8d-2462f3fda1aa.png)

**Comment by pxpm** [comment, marked as answer]
Hello @rickyviz 

Please have a look in our fields and update your code. You should be using our helper `old_empty_or_null()` with the coalescing chain to properly get field values.

**Reply by rickyviz** [reply]
i cant find where is fields using this helper, can give example, or file location
![image](https://user-images.githubusercontent.com/89818695/167267295-f4bf18e2-a54e-4caf-b2f2-287834698c01.png)
